pkg/sdk/common_types.go

```diff
@@ -1,5 +1,10 @@
 package sdk
 
+type LimitPagination struct {
```

Comment:
because it is specifically used for limit. actually this is used for show shares: https://docs.snowflake.com/en/sql-reference/sql/show-shares, so i can remove it and add it to that PR where it might make more sense. Most resources "limit" only a number of rows, but this one also has "from" which is used for pagination purposes of the limit function. Hence "LimitPagination"